.github/workflows/api-update.yaml

```diff
@@ -14,7 +13,6 @@ on:
         - "https://raw.githubusercontent.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/main/openapi/kas-fleet-manager.yaml"
         - "https://raw.githubusercontent.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/main/openapi/connector_mgmt.yaml"
         - "https://raw.githubusercontent.com/bf2fc6cc711aee1a0c2a/srs-fleet-manager/main/core/src/main/resources/srs-fleet-manager.json"
```

Comment:
Here we can either list out all repos OAS files or get rid of `inputs` altogether and just use a script to pull all OAS files. 
as this is currently, it only gives us the option for the 3 listed OAS files

Reply:
Having the full list I think is better right because then we can select the one we want for a manual run.